example/src/components/StandardTextInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { AdditionalColors, BrandColors, TextColors } from '@colors';
+import { TextInputTextStyle } from '@components/Typo';
+import React, { useState } from 'react';
+import { StyleSheet, TextInput } from 'react-native';
+
+const StandardTextInputStyles = StyleSheet.create({
+  main: {
+    width: '100%',
+    height: 56,
+    borderRadius: 40,
+    borderStyle: 'solid',
+    borderWidth: 2,
+    backgroundColor: AdditionalColors.white,
+    paddingLeft: 16,
+  },
+  active: {
+    color: TextColors.darkText,
+  },
+  notActive: {
+    color: AdditionalColors.grey80,
+    borderColor: AdditionalColors.grey60,
+  },
+  offFocus: {
+    borderColor: BrandColors.darkBlue100,
+  },
+  onFocus: {
+    borderColor: BrandColors.seaBlue80,
+  },
+});
+
+type OnChangeTextType = (text: string) => void;
+
+type StandardTextInputProps = {
+  placeholder?: string;
+  value?: string;
+  editable?: boolean;
+  onChangeText?: OnChangeTextType;
+};
+
+export const StandardTextInput = ({
+  placeholder = '',
+  value = undefined,
+  editable = true,
+  onChangeText = () => {},
+}: StandardTextInputProps) => {
+  const [focusStyle, setFocusStyle] = useState(
+    StandardTextInputStyles.offFocus
+  );
+  const placeholderTextColor = AdditionalColors.grey80;
+  const fontStyle = TextInputTextStyle.body;
+
+  const onFocus = () => {
+    setFocusStyle(StandardTextInputStyles.onFocus);
+  };
+
+  const offFocus = () => {
+    setFocusStyle(StandardTextInputStyles.offFocus);
+  };
+
+  const GetStyleForTextInput = () => {
+    if (editable) {
+      return [
+        StandardTextInputStyles.main,
+        StandardTextInputStyles.active,
+        focusStyle,
+        fontStyle,
+      ];
+    }
+
+    return [
+      StandardTextInputStyles.main,
+      StandardTextInputStyles.notActive,
+      fontStyle,
+    ];
+  };
+
+  return (
+    <TextInput
```

Comment:
On my Android device the cursor and the selection is dark green and that doesn't look nice. Use `selectionColor` and `cursorColor` props to set it to the correct color (dark blue?)

We may need to upgrade RN version for it to look good though

Reply:
The handles color is broken even on the newest RN version 😅 Apparently [this](https://github.com/facebook/react-native/pull/31007) didn't fix it.
Maybe try adding it on the native side like here (in `styles.xml`): https://stackoverflow.com/questions/29795197/how-to-change-color-of-edittext-handles
If you've never done anything like this you can always ask me for help

Reply:
I think this should work okey now.